motoman_driver/launch/io_relay.launch

```diff
@@ -0,0 +1,12 @@
+<!-- 
+  Wrapper launch file for I/O relay node.
+-->
+<launch>
+	<!-- IP of robot (or PC running simulation) -->
+	<arg name="robot_ip" doc="IP of controller" />
+
+	<!-- put them on the parameter server -->
+	<param name="robot_ip_address" type="str" value="$(arg robot_ip)" />
```

Comment:
Suggestion: make this a private parameter of the node instead.

I know that's not how the trajectory streamer and the state node work, but it would be better.

Reply:
Hm, on second thought: perhaps changing this now (as in: this late) would not be a good idea.

@marip8: thoughts?

Reply:
For consistency, we should probably leave this the same as the trajectory streamer. I would prefer privately namespaced parameters, though. Maybe we could make a separate PR that changes both the trajectory streamer and IO relay nodes together to pull the required parameters from a private namespace. It would touch a large number of files because we would need to modify the launch files for all robots in this and the experimental repository to load the parameters correctly